FireRender.Maya.Src/FireRenderObjects.cpp

```diff
@@ -1646,6 +1645,7 @@ void FireRenderMesh::GetShapes(std::vector<frw::Shape>& outShapes)
 
 	if (mainMesh == nullptr)
 	{
+#ifdef MESH_RELOAD_REFERENCE_DEBUG
```

Comment:
please remove this debug code under MESH_RELOAD_REFERENCE_DEBUG in all occurrences
you can make some utility code in FireRenderUtils which opens logginfFile, writes some output passed as a param and close it.